main.c

```diff
@@ -550,6 +552,10 @@ int main(int argc, char **argv)
             params.tfo = 1;
             break;
             
+        case 'L':
+            params.auto_level = atoi(optarg);
```

Comment:
`sscanf(optarg, "%d", &params.auto_level);` would be better and more error proof.

You can't check if `atoi("0")` is returned successfully parsed zero or error. It does not set errno.

Reply:
In general I prefer `strtol`, as here:
https://github.com/hufrea/byedpi/blob/1d6c0a132e7b1e788711dc4c3b502a27621347c4/main.c#L775-L777
This option is not documented yet (it is not in the help), it may be renamed and will not have arguments. while I am testing I use it to quickly disable the proposed behavior.